plugins/callback/opentelemetry.py

```diff
@@ -0,0 +1,374 @@
+# (C) 2021, Victor Martinez <VictorMartinezRubio@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    author: Victor Martinez (@v1v)  <VictorMartinezRubio@gmail.com>
+    name: opentelemetry
+    type: notification
+    short_description: create distributed traces with OpenTelemetry.
+    version_added: historical
+    description:
+      - This callback create distributed traces for each ansible task with OpenTelemetry.
+    options:
+      include_setup_tasks:
+        name: Include setup tasks
+        default: True
+        description: Should the setup tasks be included in the distributed traces
+        env:
+          - name: INCLUDE_SETUP_TASKS
+      hide_task_arguments:
+        name: Hide the arguments for a task
+        default: False
+        description: Hide the arguments for a task
+        env:
+          - name: HIDE_TASK_ARGUMENTS
+      service_name:
+        name: Set the service name.
+        default: ansible
+        description: Hide the arguments for a task
+        env:
+          - name: OTEL_SERVICE_NAME
+      otel_exporter:
+        name: Use the OTEL exporter and its environment variables. https://opentelemetry-python.readthedocs.io/en/latest/exporter/otlp/otlp.html
+        default: False
+        description: Use the OTEL exporter and its environment variables. https://opentelemetry-python.readthedocs.io/en/latest/exporter/otlp/otlp.html
```

Comment:
```suggestion
        description: Use the OTEL exporter and its environment variables. U(https://opentelemetry-python.readthedocs.io/en/latest/exporter/otlp/otlp.html)
```
Even better would be to use link titles. See https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation for more info.